packages/web-components/src/components/link-list/link-list-item.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Style schemes
+ */
+export enum STYLE_SCHEME {
+  /**
+   * Default style
+   */
+  DEFAULT = '',
+
+  /**
+   * Link List variant
+   */
+  LINK_LIST = 'link-list',
+}
```

Comment:
I should have mentioned earlier (my apologies), can we define this enum in `link-with-icon.ts` and/or `card-link.ts`? We need to define the corresponding properties to one or both of those, too, like:

```typescript
@property({ reflect: true, attribute: 'style-scheme' })
styleScheme = STYLE_SCHEME.DEFAULT;
```